vividus-plugin-mobitru/src/main/java/org/vividus/mobitru/client/MobitruFacade.java

```diff
@@ -31,6 +31,32 @@ public interface MobitruFacade
      */
     String takeDevice(DesiredCapabilities desiredCapabilities) throws MobitruOperationException;
 
+    /**
+     * Start the screen recording on the device with specified UDID.
+     *
+     * @param deviceId The UDID of the device
+     * @throws MobitruOperationException In case of any issues during start the recording.
+     */
+    void startDeviceScreenRecording(String deviceId) throws MobitruOperationException;
+
+    /**
+     * Stop the screen recording on the device with specified UDID.
+     *
+     * @param deviceId The UDID of the device
+     * @return The ID of the recording artifact.
+     * @throws MobitruOperationException In case of any issues during start the recording.
+     */
+    String stopDeviceScreenRecording(String deviceId) throws MobitruOperationException;
+
+    /**
+     * Stop the screen recording on the device with specified UDID.
+     *
+     * @param recordingId The ID of the recording artifact
+     * @return Binary content of the recording artifact.
+     * @throws MobitruOperationException In case of any issues during start the recording.
+     */
+    byte[] downloadDeviceScreenRecording(String recordingId) throws MobitruOperationException;
```

Comment:
do you mean to add `bute[] content` field to org.vividus.mobitru.client.model.ScreenRecording and then specify it in the facade method?

Reply:
I'm not sure about Mobitru REST API and its mapping to model `ScreenRecording`, probably there should be another POJO

Reply:
but downloadDeviceScreenRecording API call just returns a byte array.
how this another POJO should look like in this case ?

Reply:
just create a new object in the code from the results of 2 API calls
```java
public record ScreenRecording(String recordingId, byte[] content)
{
}
```

Reply:
do you mean that I should have 2 screen recording classes?
because, right now, I already have org.vividus.mobitru.client.model.ScreenRecording, which is using as DTO for result of stopRecording API and it contains recordingId only.

Reply:
done